src/components/entity-panel/sections/tactic/TheTacticListContent.vue

```diff
@@ -2,55 +2,85 @@
   <span v-if="!items.length" class="caption px-2" v-text="$t('tactic.noTacticsFound')"/>
   <v-treeview
     v-else
-    v-model="activeElements"
+    :active="[currentTacticId]"
     :items="items"
-    :open="open"
     class="pa-0"
     dense
     open-on-click
     activatable
+    @update:active="switchTactic($event)"
   >
     <template v-slot:prepend="{ item, isOpen }">
       <v-icon v-if="item.children">
         {{ isOpen ? 'fa-folder-open' : 'fa-folder' }}
       </v-icon>
-      <v-avatar
-        v-else
-        size="30"
-        class="ml-0 pl-0"
-      >
-        <img :src="item.tactic.map.icon">
-      </v-avatar>
+        <v-badge
+          v-else
+          icon="fa-bookmark"
+          :value="item.tactic.isPinned"
+          offset-y="15"
+          left
+          overlap
+        >
+          <v-avatar
+            size="30"
+            class="ml-0 pl-0"
+          >
+             <img :src="item.tactic.map.icon">
+           </v-avatar>
+        </v-badge>
     </template>
     <template v-slot:label="{ item }">
       <span
         v-if="!item.children"
         class="d-flex justify-space-between align-center"
-        @click="switchTactic(item.id)"
+        :title="item.name"
```

Comment:
bind under v-if

Reply:
This is fine :)
https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended